src/modules/transaction/manager/blockDetailsTransactionManger.js

```diff
@@ -1,21 +1,8 @@
 /* istanbul ignore file */
 import { compose } from 'redux';
 import { withTranslation } from 'react-i18next';
-import withData from 'src/utils/withData';
 import BlockDetailsTransactions from '../components/BlockDetailsTransactions/BlockDetailsTransactions';
-import { getTransactions } from '../api';
 
 export default compose(
```

Comment:
@soroushm If `withTranslation` is the only HOC used, then there's no need for `compose`

Reply:
We don't even need translation here